tests/integration/test_structured_logging_json/test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+import json
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance("node", main_configs=["configs/config_json.xml"])
+
+
+@pytest.fixture(scope="module")
+def start_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def get_log_array(logs):
+    log_array = []
+    temp_log = ""
+    for i in range(0, len(logs)):
+        temp_log += logs[i]
+        if logs[i] == "}":
+            log_array.append(temp_log)
+            temp_log = ""
+    return log_array
+
+
+def is_json(log_json):
+    try:
+        json.loads(log_json)
+    except ValueError as e:
+        return False
+    return True
+
+
+def test_structured_logging_json_format(start_cluster):
+    node.query("SELECT 1")
+
+    logs = node.grep_in_log(" ")
+    log_array = get_log_array(logs)
+    result = True
+    for i in range(0, len(log_array)):
+        temporary_result = is_json(log_array[i])
+        result &= temporary_result
+        # we will test maximum 5 logs
+        if i >= min(4, len(log_array) - 1):
+            break
+    assert result
```

Comment:
Is it true that we should be able to easily convert logs to JSON right after splitting them into separate lines?
Also `get_log_array` can add exactly 0 lines and the test will pass without actually testing the JSON formatting.
```suggestion
    logs = node.grep_in_log("").split('/n')
    length = min(10, len(logs))

    for i in range(0, length):
        assert is_json(logs[i]), f"Failed to convert to JSON: {logs[i]}"

```